.github/workflows/fuzzing.yaml

```diff
@@ -0,0 +1,24 @@
+name: Fuzzing v3rpc
+on: [push, pull_request]
+jobs:
+  fuzzing:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+          Target: [FuzzRangeRequest, FuzzPutRequest, FuzzDeleteRangeRequest]
+    env:
+      BASE_PATH: server/etcdserver/api/v3rpc
+      FUZZ_TIME: 300s
+      FUZZ_TARGET: ${{matrix.Target}}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: "1.19.1"
+    - run: date
+    - run: GOARCH=amd64 CPU=4 ./scripts/fuzzing.sh
```

Comment:
Could you add a fuzz command to Makefile?

Reply:
Updated Makefile with the `fuzz` command

Reply:
Can we call the makefile from here?